### PR TITLE
Example native bidder

### PR DESCRIPTION
**Pull request by valdo404**
Create an example bidder that handles native bid requests.

### CODE REVIEW

File: openrtb-examples/src/main/scala/com/powerspace/example/BidderApp.scala [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+import com.google.openrtb._
+import com.powerspace.openrtb.Bidder
+import monix.eval.Task
+import monix.execution.Scheduler
+
+import scala.concurrent.Future
+import scala.util.{Failure, Random, Success}
+
+/**
+  * @todo maybe create a simple test?
```
Comment (rlebran) [inline openrtb-examples/src/main/scala/com/powerspace/example/BidderApp.scala:24]: Please remove those todo before merging

File: build.sbt [hunk 2]
```diff
@@ -38,6 +38,10 @@ lazy val bidswitchJson = Project(id = "bidswitch-json", base = file("bidswitch-j
   .dependsOn(bidswitchModel % "compile->compile;test->test", openRtbJson % "compile->compile;test->test")
   .settings(testDependencies: _*)
 
+// examples
+lazy val examples = Project(id = "openrtb-examples", base = file("openrtb-examples"))
+  .dependsOn(openRtbJson % "compile->compile;test->test")
```
Comment (rlebran) [inline build.sbt:47]:
```suggestion
  .dependsOn(openRtbJson % "compile->compile;test->test")
  .settings(skip in publish := true)
```

**Review by rlebran — COMMENTED** [review]
I will merge the changes I made on master branch for the release process back to develop. I will rebase your branch when it's done done

**Comment by rlebran** [comment]
Looks good to me. I would like to have an example client to test the example bidder as well if you can provide one.

File: openrtb-examples/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+Build you own bidder with Scala OpenRTB
+[???]
+Real-time bidding refers to the buying and selling of online ad impressions through real-time auctions that occur in the time it takes a webpage to load. 
+We're talking about a whole bidding process going on in order to choose the right ad to display in just a bunch of milliseconds. Quite impressive, isn't it? 
+Yes, it is. But that's what happens behind the scenes while visiting almost any web page out there.
+[Description of  exchange/ bidder/SSP/DSP]
+Photo by John Jackson on UnsplashToday we're going to build a full-fledged bidder capable of accepting a bid request and returning a number of bids in the form of bid response. 
+[it will be written using Tagless plus Scala OpenRTB]
+Scala OpenRTB is a full Scala library that provides protobuf bindings as well as JSON serialization for OpenRTB-based entities in a functional fashion.
+Scala OpenRTB is currently deployed in production at Powerspace, hence the project will be maintained for a good while.
```
Comment (emanuele-pirro) [inline openrtb-examples/README.md]: Pasted the whole article draft in here. Needs to be removed or it's intended?

File: openrtb-examples/src/main/scala/com/powerspace/example/BidderApp.scala [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+    bids.map{
+      case Nil => None
+      case bids@_ => Some(withProtocol(BidResponse(
+        id = bidRequest.id,
+        seatbid = Seq(SeatBid(bid = bids)))),
+      )}
+  }
+
+  private def bidsForImpression(impression: BidRequest.Imp): Observable[Bid] = {
```
Comment (emanuele-pirro) [inline openrtb-examples/src/main/scala/com/powerspace/example/BidderApp.scala:80]: Just a few lines of comment above each functions describe what we're actually doing

**Pull request merged**